### PR TITLE
MTT: Prevent the route for Additional Ingress

### PR DESCRIPTION
**Pull request by pbajjuri20**
https://issues.redhat.com/browse/OSSM-5026

### CODE REVIEW

**Review by monroe-dylan-m — APPROVED** [review]
LGTM

File: pkg/tests/ossm/route_prevent_additional_ingress_test.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/shell"
+	. "github.com/maistra/maistra-test-tool/pkg/util/test"
+	"github.com/maistra/maistra-test-tool/pkg/util/version"
+)
+
+func TestRoutePreventAdditionalIngress(t *testing.T) {
+	NewTest(t).Id("T48").Groups(Full).MaxVersion(version.SMCP_2_5).Run(func(t TestHelper) {
+
+		ER := "extra-routes"
```
Comment (fjglira) [inline pkg/tests/ossm/route_prevent_additional_ingress_test.go:17]: This should be more explicit, for example: `extraRoutesNS`. With this when you are reading will be more clear for what will be used

File: pkg/tests/ossm/route_prevent_additional_ingress_test.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+		t.Cleanup(func() {
+			oc.DeleteFromString(t, ER, smcp_additionalIngress)
+			oc.DeleteNamespace(t, "extra-routes")
```
Comment (fjglira) [inline pkg/tests/ossm/route_prevent_additional_ingress_test.go:21]: Here, better to use the same variable defined above:
`oc.DeleteNamespace(t, extraRoutesNS)`

**Review by fjglira — COMMENTED** [review]
I know that is already merged but I add two comments that I think can improve the reading of the code

File: pkg/tests/ossm/route_prevent_additional_ingress_test.go (hunk 2 above)
Comment (pbajjuri20) [reply pkg/tests/ossm/route_prevent_additional_ingress_test.go:21]: @fjglira : Thanks for checking. Will update it.

**Pull request merged**